data/datapackage.yaml

```diff
@@ -0,0 +1,338 @@
+          constraints:
+            required: true
+        - name: alternate_title
+          type: any
+          description: '' # This means something specific, right?
+        - name: work_year
+          type: integer
+          description: Year in which the work was published
+        - name: copyright_year
+          type: integer
+          description: Year in which the work was copyrighted
+        - name: print_date
+          type: any
+          description: Print date of the work
+        - name: authors
+          type: string
+          description: Authors of the work
+        - name: contributors
+          type: any
+          description: Known contributors to the work (e.g translators or editors)
+        - name: publisher
+          type: string
+          description: Publisher of the work
+        - name: pub_place
+          type: string
+          description: City in which the work was published
+        - name: is_extant
+          type: boolean
+          description: Work exists in Derrida's library # ?
+          constraints:
+            required: true
+        - name: is_annotated
+          type: boolean
+          description: The instance was annotated
+          constraints:
+            required: true
+        - name: is_translation
+          type: boolean
+          description: The work is a translation
+          constraints:
+            required: true
+        - name: has_dedication
+          type: boolean
+          description: '' # ? 
+          constraints:
+            required: true
+        - name: has_insertions
+          type: boolean
+          description: The instance contains insertions
+          constraints:
+            required: true
+        - name: copy
+          type: string
+          description: '' # ?
+          constraints:
+            pattern: A|B
+        - name: dimensions
+          type: any
+          description: Dimensions of the instance in Derrida's library
+        - name: subjects
+          type: string
+          description: The subjects covered in the work
+          # ? Was this according to some formal or external system?
+        - name: languages
+          type: string
+          description: The languages covered in the work
+        - name: journal_title
+          type: any
+          description: The name of the journal in which the journal article was published
+        - name: collected_work_title
+          type: any
+          description: The title of the work in which the given item is collected
+        - name: collected_work_uri
+          type: string
+          format: uri
+          description: The URI of the work in which the given item is collected
+        - name: start_page
+          type: any
+          description: Start page of the work, often within a collected work # ?
+        - name: end_page
+          type: any
+          description: End page of the work, often within a collected work # ?
```

Comment:
I think these would only apply to journal articles and book sections

Reply:
✅ f2206137afad30f73958d34b9fed312ec56ecb06